scopes/mdx/readme/readme.main.runtime.ts

```diff
@@ -0,0 +1,25 @@
+import { EnvsAspect, EnvsMain } from '@teambit/envs';
+import { MainRuntime } from '@teambit/cli';
+import MDXAspect, { MdxEnv, MDXMain } from '@teambit/mdx';
+import DocsAspect, { DocsMain } from '@teambit/docs';
+
+import { ReadmeAspect } from './readme.aspect';
+import { ReadmeEnv } from './readme.env';
+
+export class ReadmeMain {
+  static runtime = MainRuntime;
+  static dependencies = [EnvsAspect, MDXAspect, DocsAspect];
+  constructor(protected env: ReadmeEnv & MdxEnv, private docs: DocsMain) {}
+  icon() {
+    // TODO: Add icon for aspect
+    return 'https://static.bit.dev/extensions-icons/default.svg';
```

Comment:
what icon do we want for readme components? an aspect icon or mdx icon? (I wouldn't expect it to be an aspect icon as it will with this implementation)